code/modules/reagents/reagent_containers/borghydro.dm

```diff
@@ -69,6 +73,14 @@
 	var/datum/reagents/R = reagent_list[reagent_list.len]
 	R.add_reagent(reagent, 30)
 
+/obj/item/reagent_containers/borghypo/proc/refill_borghypo(var/datum/reagents/RG, var/reagent_id)
+	if(RG.total_volume < RG.maximum_volume)
+		var/mob/living/silicon/robot/R = loc
```

Comment:
best to pass this as a parameter. Can just be called `R` or `owner`. Whatever you think fits best

Reply:
done